### DISCUSSION TITLE
Option to archive page URL and links appearing in content/selected text

### DISCUSSION THREAD

**Discussion by Zebrazilla** [coddingtonbear/obsidian-web#21] [Ideas]
Sometimes content disappear over time.

The plugin [Obsidian Link Archive](https://github.com/tomzorz/obsidian-link-archive) by tomzorz can automate archiving an external link on [Archive.org's Wayback Machine](https://web.archive.org/) and append the archived URL.

It would be very useful if Obsidian Web could do this as well, i.e. archive the page URL which is being sent to Obsidian, and also archive any links appearing in _content_ or _selected text_, before it is sent to Local REST API.

**Comment by coddingtonbear** [comment]
The configurability of this tool is already probably too flexible (read: complex) in order to allow for every user's idiosyncratic desires from their note taking tool.  It seems like adding such a feature would just multiply that complexity; so I'm not likely to personally add this.

If you wanted to take a stab at it, though, I'd be interested in hearing how you might imagine the UI for this working from a technical and UI/UX perspective before you put a PR together and would be glad to help nudge you in the right direction whenever you ran into questions.

Just to be clear: what you've suggested here does seem like a genuinely useful feature that's in the spirit of the kind of problem this plugin is intended to solve, it's just that I can't personally commit the time & can't imagine how to make that intuitive enough for people to use, flexible enough to allow folks some avenues for satisfying their idiosyncratic needs, and robust enough that it would work enough (roughly: all) of the time.
